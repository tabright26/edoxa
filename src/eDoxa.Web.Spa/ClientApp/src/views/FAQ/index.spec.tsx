import React from "views/Faq/node_modules/react";
import renderer from "views/Faq/node_modules/react-test-renderer";
import { Provider } from "views/Faq/node_modules/react-redux";
import Faq from ".";

it("renders without crashing", () => {
  // Arrange
  const store: any = {
    getState: () => {},
    dispatch: action => {},
    subscribe: () => {}
  };

  // Act
  const tree = renderer
    .create(
      <Provider store={store}>
        <Faq />
      </Provider>
    )
    .toJSON();

  // Assert
  expect(tree).toMatchSnapshot();
});
