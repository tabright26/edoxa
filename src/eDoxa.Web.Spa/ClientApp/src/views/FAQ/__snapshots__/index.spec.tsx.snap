// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <h5
    className="text-uppercase my-4"
  >
    FREQUENTLY ASKED QUESTIONS
  </h5>
  <div
    id="accordion"
  >
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-0"
      >
        <button
          aria-controls="collapse-0"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            Why EDOXA WAS CREATED?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-0"
        className="collapse"
        data-parent="#accordion"
        id="collapse-0"
        style={Object {}}
      >
        <div
          className="card-body"
        >
          <p
            className="mb-0"
          >
            We were frustrated as gamers to have a limited number of options to be successful. eDoxa was born from the sear to help unlock every gamer’s potential so that they can be proud to live from their eSport dream. Thus, everything we do and every new feature we develop is required to push our users in their path to success. We believe so strongly in our mission that we will give throughout the year 5%-10% of our revenue back in form of tokens to all participants that won’t make money in our Challenges. This will give them the opportunity to always try and to never lose hope in making it in this cutthroat world.
             
          </p>
        </div>
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-1"
      >
        <button
          aria-controls="collapse-1"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT TO EXPECT IN THE FUTURE FOR EDOXA?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-1"
        className="collapse"
        data-parent="#accordion"
        id="collapse-1"
        style={Object {}}
      >
        <div
          className="card-body"
        >
          <p
            className="mb-0"
          >
            We are only at the beginning of our journey to become the #1 enabler of eSport talent in the world. We have a robust road map for the product, and this will change based on the feedback we get from you. Please contact us here or you can contact us
             
            <a
              href="https://discord.gg/SvBUH5Y"
            >
              here
            </a>
             at
             
            <a
              href="emailto:support@edoxa.gg"
            >
              support@edoxa.gg
            </a>
            .
          </p>
        </div>
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-2"
      >
        <button
          aria-controls="collapse-2"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS EDOXA?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-2"
        className="collapse"
        data-parent="#accordion"
        id="collapse-2"
        style={Object {}}
      >
        <div
          className="card-body"
        >
          <p
            className="mb-0"
          >
            We are only at the beginning of our journey to become the #1 enabler of eSport talent in the world. We have a robust road map for the product, and this will change based on the feedback we get from you. Please contact us here or you can contact us
             
            <a
              href="https://discord.gg/SvBUH5Y"
            >
              here
            </a>
             at
             
            <a
              href="emailto:support@edoxa.gg"
            >
              support@edoxa.gg
            </a>
            .
          </p>
        </div>
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-3"
      >
        <button
          aria-controls="collapse-3"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I SIGN UP?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-3"
        className="collapse"
        data-parent="#accordion"
        id="collapse-3"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-4"
      >
        <button
          aria-controls="collapse-4"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I PARTICIPATE TO CHALLENGES ON EDOXA WITHIN 5 MINUTES?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-4"
        className="collapse"
        data-parent="#accordion"
        id="collapse-4"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-5"
      >
        <button
          aria-controls="collapse-5"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I RESET MY PASSWORD?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-5"
        className="collapse"
        data-parent="#accordion"
        id="collapse-5"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-6"
      >
        <button
          aria-controls="collapse-6"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT HAPPENS IF MY OPPONENT OR I GET DISCONNECTED DURING AN ONLINE MATCH?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-6"
        className="collapse"
        data-parent="#accordion"
        id="collapse-6"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-7"
      >
        <button
          aria-controls="collapse-7"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS A CHALLENGE?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-7"
        className="collapse"
        data-parent="#accordion"
        id="collapse-7"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-8"
      >
        <button
          aria-controls="collapse-8"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            CAN WE REGISTER TO MULTIPLE CHALLENGES?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-8"
        className="collapse"
        data-parent="#accordion"
        id="collapse-8"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-9"
      >
        <button
          aria-controls="collapse-9"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT GAME MODE NEEDS TO BE PLAYED FOR LEAGUE OF LEGENDS CHALLENGES?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-9"
        className="collapse"
        data-parent="#accordion"
        id="collapse-9"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-10"
      >
        <button
          aria-controls="collapse-10"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT’S A TIMELINE [STATE]?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-10"
        className="collapse"
        data-parent="#accordion"
        id="collapse-10"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-11"
      >
        <button
          aria-controls="collapse-11"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS YOUR SCORING LEGEND?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-11"
        className="collapse"
        data-parent="#accordion"
        id="collapse-11"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-12"
      >
        <button
          aria-controls="collapse-12"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DOES THE SCORECARD WORKS?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-12"
        className="collapse"
        data-parent="#accordion"
        id="collapse-12"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-13"
      >
        <button
          aria-controls="collapse-13"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT TYPE OF CHALLENGES DO YOU OFFER?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-13"
        className="collapse"
        data-parent="#accordion"
        id="collapse-13"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-14"
      >
        <button
          aria-controls="collapse-14"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            IS THERE A MINIMUM NUMBER OF GAMES I MUST PLAY TO GET PAID?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-14"
        className="collapse"
        data-parent="#accordion"
        id="collapse-14"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-15"
      >
        <button
          aria-controls="collapse-15"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT ARE THE ENTRY FEES FOR A CHALLENGE?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-15"
        className="collapse"
        data-parent="#accordion"
        id="collapse-15"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-16"
      >
        <button
          aria-controls="collapse-16"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW MANY GAMES DOES EDOXA SUPPORTS?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-16"
        className="collapse"
        data-parent="#accordion"
        id="collapse-16"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-17"
      >
        <button
          aria-controls="collapse-17"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW LONG DOES IT TAKE TO SEE MY SCORE IN A CHALLENGE?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-17"
        className="collapse"
        data-parent="#accordion"
        id="collapse-17"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-18"
      >
        <button
          aria-controls="collapse-18"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS THE PAYOUT STRUCTURE?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-18"
        className="collapse"
        data-parent="#accordion"
        id="collapse-18"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-19"
      >
        <button
          aria-controls="collapse-19"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHO IS ELIGIBLE TO PLAY ON EDOXA?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-19"
        className="collapse"
        data-parent="#accordion"
        id="collapse-19"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-20"
      >
        <button
          aria-controls="collapse-20"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW TO ADD A CREDIT CARD?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-20"
        className="collapse"
        data-parent="#accordion"
        id="collapse-20"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-21"
      >
        <button
          aria-controls="collapse-21"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I DEPOSIT MONEY IN MY WALLET?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-21"
        className="collapse"
        data-parent="#accordion"
        id="collapse-21"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-22"
      >
        <button
          aria-controls="collapse-22"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I ADD MY BANK ACCOUNT?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-22"
        className="collapse"
        data-parent="#accordion"
        id="collapse-22"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-23"
      >
        <button
          aria-controls="collapse-23"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            HOW DO I WITHDRAW MONEY?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-23"
        className="collapse"
        data-parent="#accordion"
        id="collapse-23"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-24"
      >
        <button
          aria-controls="collapse-24"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT ARE THE REQUIREMENTS TO WITHDRAW MONEY?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-24"
        className="collapse"
        data-parent="#accordion"
        id="collapse-24"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-25"
      >
        <button
          aria-controls="collapse-25"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT ARE THE REQUIREMENTS TO DEPOSIT MONEY?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-25"
        className="collapse"
        data-parent="#accordion"
        id="collapse-25"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-26"
      >
        <button
          aria-controls="collapse-26"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHO IS YOUR PLAYMENT PROVIDER?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-26"
        className="collapse"
        data-parent="#accordion"
        id="collapse-26"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-27"
      >
        <button
          aria-controls="collapse-27"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            IS EDOXA LEGAL?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-27"
        className="collapse"
        data-parent="#accordion"
        id="collapse-27"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-28"
      >
        <button
          aria-controls="collapse-28"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS SKILLED BASED GAMING?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-28"
        className="collapse"
        data-parent="#accordion"
        id="collapse-28"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-29"
      >
        <button
          aria-controls="collapse-29"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            DOES MY STATE OR PROVINCE ALLOW SKILL-BASED GAMING?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-29"
        className="collapse"
        data-parent="#accordion"
        id="collapse-29"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-30"
      >
        <button
          aria-controls="collapse-30"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS THE LEGAL AGE TO PLAY ON eDoxa.gg?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-30"
        className="collapse"
        data-parent="#accordion"
        id="collapse-30"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-31"
      >
        <button
          aria-controls="collapse-31"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            ARE YOU A PARTNER OR RIOT?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-31"
        className="collapse"
        data-parent="#accordion"
        id="collapse-31"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-32"
      >
        <button
          aria-controls="collapse-32"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS A SMURF?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-32"
        className="collapse"
        data-parent="#accordion"
        id="collapse-32"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-33"
      >
        <button
          aria-controls="collapse-33"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT ARE THE CONSEQUENCES OF HAVING A SMURF ACCOUNT?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-33"
        className="collapse"
        data-parent="#accordion"
        id="collapse-33"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-34"
      >
        <button
          aria-controls="collapse-34"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            WHAT IS YOUR REFUND POLICY?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-34"
        className="collapse"
        data-parent="#accordion"
        id="collapse-34"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
    <div
      className="my-1 card"
    >
      <div
        className="card-header"
        id="button-35"
      >
        <button
          aria-controls="collapse-35"
          aria-expanded={false}
          aria-label={null}
          className="text-left m-0 p-0 text-muted btn btn-link btn-block"
          onClick={[Function]}
          type="button"
        >
          <h5
            className="m-0 p-0 text-uppercase"
          >
            IS eDoxa.gg SECURE?
          </h5>
        </button>
      </div>
      <div
        aria-labelledby="button-35"
        className="collapse"
        data-parent="#accordion"
        id="collapse-35"
        style={Object {}}
      >
        <div
          className="card-body"
        />
      </div>
    </div>
  </div>
</div>
`;
